app/main.py

```diff
@@ -1 +1,15 @@
-# write your code here
+from app.players.player import Player
+from app.players.elves.elf import Elf
+from app.players.dwarves.dwarf import Dwarf
+
+
+def calculate_team_total_rating(players_list: list[Player]) -> int:
+    return sum(player.get_rating() for player in players_list)
+
+
+def elves_concert(elfs_list: list[Elf]) -> None:
+    [elf.play_elf_song() for elf in elfs_list]
+
+
+def feast_of_the_dwarves(dwarves_list: list[Dwarf]) -> None:
+    [dwarf.eat_favourite_dish() for dwarf in dwarves_list]
```

Comment:
no need comprehensions here.
also, `elfs_list` or `dwarves_list` are not good names. use just `elves`, for example